backend/molgenis-emx2-beacon-v2/src/main/java/org/molgenis/emx2/beaconv2/vcfimport/VcfImport.java

```diff
@@ -0,0 +1,78 @@
+package org.molgenis.emx2.beaconv2.vcfimport;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFFileReader;
+import htsjdk.variant.vcf.VCFHeader;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.molgenis.emx2.Row;
+import org.molgenis.emx2.Table;
+
+public class VcfImport {
+
+  private File vcfTmpFile;
+  private Table genomicVariations;
+  private Table individuals;
+
+  public VcfImport(File vcfTmpFile, Table genomicVariations, Table individuals) {
+    this.vcfTmpFile = vcfTmpFile;
```

Comment:
would call it 'vcfFile' (might or might not be tempfile, doesn't matter here)

Reply:
Sure